pages/index.tsx

```diff
@@ -60,22 +60,24 @@ export default function Home({ deviceType }: Props) {
       coordinatesAndEventType?.sw_lng,
     ]
   );
-  const { error, isLoading, mutate, isValidating } = useSWR<
-    DataLite | undefined
-  >(url, dataFetcher, {
-    onLoadingSlow: () => setSlowLoading(true),
-    revalidateOnFocus: false,
-    onSuccess: (data) => {
-      if (!data) return;
+  const { error, isLoading, isValidating } = useSWR<DataLite | undefined>(
+    url,
+    dataFetcher,
+    {
+      onLoadingSlow: () => setSlowLoading(true),
+      revalidateOnFocus: false,
+      onSuccess: (data) => {
+        if (!data) return;
 
-      const transformedData = data.results ? dataTransformerLite(data) : [];
-      setMarkerData(transformedData);
-    },
-  });
+        const transformedData = data.results ? dataTransformerLite(data) : [];
+        setMarkerData(transformedData);
+      },
+    }
+  );
 
   const { setDevice } = useMapActions();
   const [remainingTime, resetThrottling] = useIncrementalThrottling(
-    mutate,
```

Comment:
Bunu kaldirmaktaki tek sikinti, cache'i invalidate etmedigimiz icin eger daha onceden ayni URL cachelendiyse sayfa refresh olana kadar ayni veriyi gosterecegiz. 

Su settingi enable edebiliriz belki https://swr.vercel.app/docs/revalidation#revalidate-on-interval 

Sonucta verinin cok hizli degismesini beklemiyoruz, 1-2 dakikada bir cache'i invalidate edebiliriz.

Reply:
Bu durumda butonu throttle etmenin de pek anlami kalmiyor, zaten kullanici istedigi kadar bassin yine veri cache'ten gelecegi icin engel olmanin pek anlami kalmiyor.

Reply:
Şu an swrın cache özelliğini kullanabildiğimizden çok emin değilim. Url query params içerisinde lat, lon verileri olduğundan ve neredeyse hiç kıpırdamadan bile zoom-in zoom out yapsa illa ki url değişiyor yani yeni bir cache key yaratmış oluyoruz.

![image](https://user-images.githubusercontent.com/4690150/217863674-655021f4-74ef-4a7c-9426-5ba603d0e0b8.png)

Reply:
@Kibyra evet, haritada cok efektif degil. Yine de ayni bolge icerisinde surekli dolasan birisi icin cache bir miktar hit alabilir ve bu durumlar icin invalidate etmiyoruz su an sayfa refresh harici.

Reply:
bu arada sıkıntı zoom out dan sonra son url den veri çekmeye çalışması doğal olarak data güncellenmiyor revalidate bu sorunu nasıl çözecek ben birşey mi kaçırıyorum burada

Reply:
Yok, hayir, cozumunuz calisiyor 👍 Ben fazla edge caseleri dusunuyor olabilirim su an.